### ISSUE TITLE
Thread '<unnamed>' panicked at 'called `Result::unwrap()`

### ISSUE THREAD

**Issue by daniellivingston** [ogham/exa#747]
Running `exa --header --long` causes this error:

```bash
$ RUST_BACKTRACE=full exa --header --long
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: SystemTimeError(6795364578.871345152s)', src/libcore/result.rs:1084:5
stack backtrace:
   0:        0x10e835ced - std::panicking::default_hook::{{closure}}::hffc69ca72fa98b80
   1:        0x10e8357bc - std::panicking::rust_panic_with_hook::h058ef423beec40ce
   2:        0x10e835f7d - std::panicking::continue_panic_fmt::h0bb2cdb45a553955
   3:        0x10e836249 - rust_begin_unwind
   4:        0x10e8c7dfa - core::panicking::panic_fmt::h2413d97d26203710
   5:        0x10e8c8399 - core::result::unwrap_failed::hef33208e3f23e07a
   6:        0x10e7fb6bd - exa::output::table::Table::display::h7d61868c2e1aa281
   7:        0x10e80e955 - <F as scoped_threadpool::FnBox>::call_box::hfeff2cdcb76e7a57
   8:        0x10e826d6e - std::sys_common::backtrace::__rust_begin_short_backtrace::hf388fc36537a8fe9
   9:        0x10e82789e - core::ops::function::FnOnce::call_once{{vtable.shim}}::hd49318dcebc5ef76
  10:        0x10e84013d - <alloc::boxed::Box<F> as core::ops::function::FnOnce<A>>::call_once::hf778f932793a5a82
  11:        0x10e840767 - std::sys::unix::thread::Thread::new::thread_start::ha4ccb7099fa6037f
  12:     0x7fff6e7ea109 - _pthread_start
[2]    66927 abort      RUST_BACKTRACE=full exa --header --long
```

## System

* OS: macOS 10.15.7
* exa: 0.9.0 (Homebrew version)

**Comment by Flakebi** [comment]
I got a similar error, it seems like there can be files with a timestamp before 1970, as the unix timestamp is signed.

In this case it seems to be the lowest value possible (i.e. 32-bit unix timestamp is -1).

Output of `ls -l`:
```python
drwxr-xr-x  2 sebi users 4096 20. Jun 2018  archetypes
drwxr-xr-x  4 sebi users 4096 27. Jan 2019  assets
drwxr-xr-x  3 sebi users 4096 27. Jan 2019  config
drwxr-xr-x  8 sebi users 4096 27. Jan 2019  content
drwxr-xr-x  3 sebi users 4096 27. Jan 2019  data
-rw-r--r--  1 sebi users  523 11. Okt 16:52 flake.lock
-rw-r--r--  1 sebi users  630 11. Okt 17:05 flake.nix
drwxr-xr-x  3 sebi users 4096 28. Jun 2018  layouts
drwxr-xr-x 20 sebi users 4096 13. Dez 1901  public
drwxr-xr-x  3 sebi users 4096 26. Dez 2018  resources
drwxr-xr-x  5 sebi users 4096  6. Feb 2019  static
drwxr-xr-x  3 sebi users 4096 20. Jun 2018  themes
```

Output of `exa -l`:
```python
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: SystemTimeError(2147483648s)', src/fs/file.rs:331:9
stack backtrace:
   0: <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt
   1: core::fmt::write
   2: std::io::Write::write_fmt
   3: std::panicking::default_hook::{{closure}}
   4: std::panicking::rust_panic_with_hook
   5: rust_begin_unwind
   6: core::panicking::panic_fmt
   7: core::result::unwrap_failed
   8: exa::fs::file::File::modified_time
   9: exa::output::table::Table::display
  10: <F as scoped_threadpool::FnBox>::call_box
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
“exa -l” terminated by signal SIGABRT (Abort)
```

**Comment by Freaky** [comment]
Should be fixed in master by #670.

**Comment by ogham** [comment]
Yes, the fix for this has been merged and will go out with the next release.
